app/models/miq_server/status_management.rb

```diff
@@ -3,14 +3,39 @@
 module MiqServer::StatusManagement
   extend ActiveSupport::Concern
 
+  def status_update
+    self.attributes = system_status
+    self.attributes = process_status
+    save!
```

Comment:
@Fryguy I actually chose this way because I didn't like merging unrelated hashes with no overlapping keys.  It seemed cleaner to set just the specific system attributes, then the process ones and save at the end.  ❓